boards.txt

```diff
@@ -24112,3 +24112,225 @@ nebulas3.menu.EraseFlash.all=Enabled
 nebulas3.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
+lionbits3.name=Lion:Bit S3 STEM Dev Board
+lionbits3.vid.0=0x303a
+lionbits3.pid.0=0x1001
+
+lionbits3.bootloader.tool=esptool_py
+lionbits3.bootloader.tool.default=esptool_py
+
+lionbits3.upload.tool=esptool_py
+lionbits3.upload.tool.default=esptool_py
+lionbits3.upload.tool.network=esp_ota
+
+lionbits3.upload.maximum_size=1310720
+lionbits3.upload.maximum_data_size=327680
+lionbits3.upload.flags=
+lionbits3.upload.extra_flags=
+lionbits3.upload.use_1200bps_touch=false
+lionbits3.upload.wait_for_upload_port=false
+
+lionbits3.serial.disableDTR=false
+lionbits3.serial.disableRTS=false
+
+lionbits3.build.tarch=xtensa
+lionbits3.build.bootloader_addr=0x0
+lionbits3.build.target=esp32s3
+lionbits3.build.mcu=esp32s3
+lionbits3.build.core=esp32
+lionbits3.build.variant=lionbits3
+lionbits3.build.board=LIONBITS3_DEV
+
+
+lionbits3.build.usb_mode=1
+lionbits3.build.cdc_on_boot=0
+lionbits3.build.msc_on_boot=0
+lionbits3.build.dfu_on_boot=0
+lionbits3.build.f_cpu=240000000L
+lionbits3.build.flash_size=4MB
+lionbits3.build.flash_freq=80m
+lionbits3.build.flash_mode=dio
+lionbits3.build.boot=qio
+lionbits3.build.boot_freq=80m
+lionbits3.build.partitions=default
+lionbits3.build.defines=
+lionbits3.build.loop_core=
+lionbits3.build.event_core=
+lionbits3.build.psram_type=qspi
+lionbits3.build.memory_type={build.boot}_{build.psram_type}
+
+
+lionbits3.menu.JTAGAdapter.default=Disabled
+lionbits3.menu.JTAGAdapter.default.build.copy_jtag_files=0
+lionbits3.menu.JTAGAdapter.builtin=Integrated USB JTAG
+lionbits3.menu.JTAGAdapter.builtin.build.openocdscript=lionbits3-builtin.cfg
+lionbits3.menu.JTAGAdapter.builtin.build.copy_jtag_files=1
+lionbits3.menu.JTAGAdapter.external=FTDI Adapter
+lionbits3.menu.JTAGAdapter.external.build.openocdscript=lionbits3-ftdi.cfg
+lionbits3.menu.JTAGAdapter.external.build.copy_jtag_files=1
+lionbits3.menu.JTAGAdapter.bridge=ESP USB Bridge
+lionbits3.menu.JTAGAdapter.bridge.build.openocdscript=lionbits3-bridge.cfg
+lionbits3.menu.JTAGAdapter.bridge.build.copy_jtag_files=1
```

Comment:
This won't work as you don't provide new configs. You need to leave the original JTAG configs from ESP32S3 Dev board.
```suggestion
lionbits3.menu.JTAGAdapter.default=Disabled
lionbits3.menu.JTAGAdapter.default.build.copy_jtag_files=0
lionbits3.menu.JTAGAdapter.builtin=Integrated USB JTAG
lionbits3.menu.JTAGAdapter.builtin.build.openocdscript=esp32s3-builtin.cfg
lionbits3.menu.JTAGAdapter.builtin.build.copy_jtag_files=1
lionbits3.menu.JTAGAdapter.external=FTDI Adapter
lionbits3.menu.JTAGAdapter.external.build.openocdscript=esp32s3-ftdi.cfg
lionbits3.menu.JTAGAdapter.external.build.copy_jtag_files=1
lionbits3.menu.JTAGAdapter.bridge=ESP USB Bridge
lionbits3.menu.JTAGAdapter.bridge.build.openocdscript=esp32s3-bridge.cfg
lionbits3.menu.JTAGAdapter.bridge.build.copy_jtag_files=1
```